admin-dev/themes/new-theme/js/pages/order/create/shipping-renderer.js

```diff
@@ -190,4 +204,22 @@ export default class ShippingRenderer {
   _hideNoCarrierBlock() {
     this.$noCarrierBlock.addClass('d-none');
   }
+
+  /**
+   * Show warning message block which warns that free shipping is already added
+   *
+   * @private
+   */
+  _showNoFreeShippingSwitchHelp() {
```

Comment:
Please don't use underscore, the merge on develop will be hard instead :(

Reply:
thank you :slightly_frowning_face: